docs/announcing-telepresence-2.html

```diff
@@ -68,7 +68,7 @@
                 <div class="dropdown">
                     <button class="dropbtn">▾ Need Help?</button>
                     <div class="dropdown-content">
-                        <a target="_blank" href="https://join.slack.com/t/datawire-oss/shared_invite/enQtMzcwMDEwMTc5ODQ3LTE1NmIzZTFmZWE0OTQ1NDc2MzE2NTkzMDAzZWM0MDIxZTVjOGIxYmRjZjY3N2M2Mjk4NGI5Y2Q4NGY4Njc1Yjg">Community Slack</a>
+                        <a target="_blank" href="https://d6e.co/slack">Community Slack</a>
```

Comment:
the OSS Slack URL is now https://a8r.io/Slack, here and it looks like this will need to be changed on many other pages